programs/mango-v4/src/accounts_ix/openbook_v2_cancel_order.rs

```diff
@@ -0,0 +1,49 @@
+use anchor_lang::prelude::*;
+
+use crate::error::*;
+use crate::state::*;
+use openbook_v2::{
+    program::OpenbookV2,
+    state::{BookSide, Market, OpenOrdersAccount},
+};
+
+#[derive(Accounts)]
+pub struct OpenbookV2CancelOrder<'info> {
```

Comment:
Am I mistaken, or do Cancel and CancelAll look identical? Let's have only one accounts struct for both, imo. (should have done that for perp too)